linux/com.system76.keyboardconfiguratorlaunchtest.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=System76 Keyboard Tester
```

Comment:
![Screenshot from 2023-04-20 15-13-28](https://user-images.githubusercontent.com/13512097/233490145-1a613620-b7cb-412b-a769-941b0dff23b4.png)

Can we give this one a different name? Otherwise they're a pain to figure out.

Reply:
I actually have no idea why it is doing that

Reply:
On cosmic they are different

Reply:
They're also the same in app library.

Reply:
Weird. Not sure how that would happen unless its makes assumptions that there aren't multiple desktop files that `Exec` the same binary (even with different arguments). Which wouldn't make much sense. Or some kind of caching (but what is cached?).

The two `.desktop` files have the right `Name=` and `Exec=` lines as installed in `/usr/share/applications`?

Reply:
They have different Exec lines, but identical Name lines
![Screenshot from 2023-04-20 15-53-17](https://user-images.githubusercontent.com/13512097/233495732-2b2cc1de-6cf5-40a6-8a55-972541e259b4.png)

Reply:
Oh wait, I was on an old commit. Updating now, and sorry if this ends up being the reason :sweat_smile:

Reply:
I just did a clean install of pop, ad installed it, they show as two different names. Something is weird with your machine @leviport

Reply:
Yep, all good now. Sorry for the false alarm. 
![Screenshot from 2023-04-20 15-56-24](https://user-images.githubusercontent.com/13512097/233496333-467199fb-2f2d-49cb-99ef-2c5c56a125a0.png)
![Screenshot from 2023-04-20 15-56-38](https://user-images.githubusercontent.com/13512097/233496334-9165a3db-d971-43bb-84f1-2187f327fd56.png)